app/templates/application.hbs

```diff
@@ -1,3 +1,3 @@
-<h2 id='title'>Welcome to Ember.js</h2>
+<h2 id='title'>Chun Lok wants to welcome you to MCAC</h2>
```

Comment:
You can leave this for now, but generally you want to keep your pull request as focused as possible. This pull request is about redirecting `/` to `/sunday`, not about updating the application layout...